classification/__main__.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import re
+
+from extractor.high_level_data_extractor import make_high_level_data_file
+from classifier.profile_data_extractor import get_classifier_data
+from pprint import pprint
+from tabulate import tabulate
+
+from multiprocessing import Pool
+
+
+def process_data_and_extract_profiles(song_id, song_file, song_output_file):
+    #make_high_level_data_file(song_file, song_output_file)
+
+    timbre, mood_relaxed, mood_party = get_classifier_data(song_output_file)
+    return song_id, timbre, mood_relaxed, mood_party
+
+
+def get_song_id(filename):
```

Comment:
Could be combined with the one in bpm_extractor like suggested earlier :)

Reply:
Has been done.